types/deneb/block_contents.yaml

```diff
@@ -0,0 +1,18 @@
+Deneb:
+  BlockContents:
+    type: object
+    description: "The required object for block production according to the Deneb CL spec."
+    properties:
+      block:
+        $ref: "./block.yaml#/Deneb/BeaconBlock"
+      blob_sidecars:
```

Comment:
I wonder if it's worth being more explicit here to indicate this is `blinded`?

Reply:
so for block content ( not blinded block content) the blob_sidecars is still blinded?

Reply:
> so for block content ( not blinded block content) the blob_sidecars is still blinded?

No. The property is of type `BlobSidecars`, not `BlindedBlobSidecars`.